mautic/mautic
# [UX] Disable "Theme" field and clear selection when "Use theme style" is set to "No" while creating a form

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This pull request addresses the user experience issue where the "Theme" field remains visible even when the "Use theme style" option is set to "No." By disabling the "Theme" field under these circumstances, we simplify the form creation process, reducing confusion for users who do not intend to use a theme.

The change will streamline the interface, making it clearer and more intuitive, especially for those who may overlook the relevance of the "Theme" field when it isn't applicable. This update benefits all users involved in form creation by enhancing usability and ensuring that only relevant options are displayed




| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/user-attachments/assets/ae33549b-08bc-4f11-809a-f8b9f1682300)                                      | ![image](https://github.com/user-attachments/assets/fea52022-6c3f-4258-91cd-25550003db89)



---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Forms page
3. Create a new form
4. Set any theme using the Theme field
5. Then set "Use theme style" to "No." 
6. Verify that the Theme selection is cleared back to "Choose one" and the field itself is now disabled with no click events.
